File: ANSv4/CatalogoLibraryVB/_recibos/DeducirItem.vb
Option Explicit On
Public Class DeducirItem

    ' Define como se llama este modulo para el control de errores
    Private Const m_sMODULENAME_ As String = "clsDeducirItem"

    'variables locales para almacenar los valores de las propiedades
    Private mvarConcepto As String
    Private mvarImporte As Single
    Private mvarPorcentaje As Boolean
    Private mvarDeduAlResto As Boolean

    Public Property DeduAlResto() As Boolean
        Set(ByVal value As Boolean)
            mvarDeduAlResto = value
        End Set
        Get
            Return mvarDeduAlResto
        End Get
    End Property

    Public Property Porcentaje() As Boolean
        Set(ByVal value As Boolean)
            mvarPorcentaje = value
        End Set

        Get
            Return mvarPorcentaje
        End Get
    End Property

    Public Property Importe() As Single
        Set(ByVal value As Single)
            mvarImporte = value
        End Set
        Get
            Return mvarImporte
        End Get
    End Property

    Public Property Concepto() As String
        Set(ByVal value As String)
            mvarConcepto = value
        End Set

        Get
            Return mvarConcepto
        End Get
    End Property

End Class

File: ANSv4/Catalogo4/_recibos/DeducirItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Catalogo._recibos
{
    public class DeducirItem
    {
        //variables locales para almacenar los valores de las propiedades
        private string mvarConcepto;
        private float mvarImporte;
        private bool mvarPorcentaje;

        private bool mvarDeduAlResto;
        public bool DeduAlResto
        {
            get { return mvarDeduAlResto; }
            set { mvarDeduAlResto = value; }
        }

        public bool Porcentaje
        {

            get { return mvarPorcentaje; }
            set { mvarPorcentaje = value; }
        }

        public float Importe
        {
            get { return mvarImporte; }
            set { mvarImporte = value; }
        }

        public string Concepto
        {
            get { return mvarConcepto; }
            set { mvarConcepto = value; }
        }
    }
}